ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -111,6 +111,8 @@ public class PeerDiscoveryController {
   private final PeerTable peerTable;
   private final Cache<Bytes, DiscoveryPeer> bondingPeers =
       CacheBuilder.newBuilder().maximumSize(50).expireAfterWrite(10, TimeUnit.MINUTES).build();
+  private final Cache<Bytes, Packet> cachedEnrRequests =
+      CacheBuilder.newBuilder().maximumSize(50).expireAfterWrite(30, SECONDS).build();
```

Comment:
The [docs on `CacheBuilder`](https://guava.dev/releases/snapshot/api/docs/com/google/common/cache/CacheBuilder.html) recommend migrating to [Caffeine](https://github.com/ben-manes/caffeine/wiki).  This is probably something to look into across the codebase - maybe for a follow-up PR.

Reply:
Is 30 seconds overly generous?  Maybe 10 seconds?